### PR TITLE
feat(compactor2): Use fixed widths for timeline display in tests

### PR DESCRIPTION
**Pull request by alamb**
While reviewing https://github.com/influxdata/influxdb_iox/pull/7079/files I was having trouble understanding some of the output because the the `min` and `max` values on the timelines were different, and thus within the same "run" a file that had min/max of 50, and 100 might be displayed as different widths depending on the other files it was with.

Thus I propose simply setting the timeline width to be a fixed size of `0 -- 1000` so that the relative widths of the files are the same.

Some of the visuals are actually less good now because they get squished too much, but I plan to improve them (by changing the min/max values in the tests) as a follow on PR

This will almost certainly conflict with https://github.com/influxdata/influxdb_iox/pull/7079 so I will merge it after that one is in.

### CODE REVIEW

File: compactor2/src/components/files_split/non_overlap_split.rs [hunk 1]
```diff
@@ -193,9 +193,9 @@ mod tests {
         ---
         - initial
         - "L1, all files 1b                                                                                    "
-        - "L1.13[600,700]                                                                    |-----L1.13-----| "
```
Comment (alamb) [inline compactor2/src/components/files_split/non_overlap_split.rs:196]: This shows the difference pretty clearly -- previously the display width was dynamically chosen based on the min/max values of the files. Now the display width is a fixed `1000`

File: compactor2/tests/layouts/core.rs [hunk 2]
```diff
@@ -164,33 +164,33 @@ async fn l1_with_overlapping_l0() {
...
+    - "L1.1[50,100] 10mb   |L1.1|                                                                          "
+    - "L1.2[100,150] 10mb      |L1.2|                                                                      "
     - "**** Simulation run 0, type=compact. 6 Input Files, 10.02mb total:"
     - "L0                                                                                                  "
-    - "L0.7[260,310] 5kb                                                               |------L0.7-------| "
-    - "L0.6[230,280] 5kb                                                    |------L0.6-------|            "
-    - "L0.5[200,250] 5kb                                         |------L0.5-------|                       "
-    - "L0.4[170,220] 5kb                             |------L0.4-------|                                   "
-    - "L0.3[140,190] 5kb                  |------L0.3-------|                                              "
+    - "L0.7[260,310] 5kb               |L0.7|                                                              "
```
Comment (alamb) [inline compactor2/tests/layouts/core.rs:177]: this shows the improvement where the intermediate output files are not shown using the full width, but the same scale as the input files secton

File: compactor2/tests/layouts/core.rs [hunk 3]
```diff
@@ -20,40 +20,40 @@ async fn all_overlapping_l0() {
...
-    - "L0.8[100,200]       |-------------------------------------L0.8-------------------------------------|"
-    - "L0.7[100,200]       |-------------------------------------L0.7-------------------------------------|"
-    - "L0.6[100,200]       |-------------------------------------L0.6-------------------------------------|"
-    - "L0.5[100,200]       |-------------------------------------L0.5-------------------------------------|"
-    - "L0.4[100,200]       |-------------------------------------L0.4-------------------------------------|"
-    - "L0.3[100,200]       |-------------------------------------L0.3-------------------------------------|"
-    - "L0.2[100,200]       |-------------------------------------L0.2-------------------------------------|"
-    - "L0.1[100,200]       |-------------------------------------L0.1-------------------------------------|"
+    - "L0, all files 9mb                                                                                        "
+    - "L0.1[0,1000]             |-------------------------------------L0.1-------------------------------------|"
```
Comment (alamb) [inline compactor2/tests/layouts/core.rs:24]: I changed the tests so the full visual range is used (the meaning is preserved)

File: compactor2/src/components/files_split/non_overlap_split.rs [hunk 4]
```diff
@@ -238,16 +238,16 @@ mod tests {
...
+        - "L0.1[450,620]                |---L0.1----|                                                               "
+        - "L0.3[800,900]                                            |-L0.3-|                                        "
+        - "L1, all files 1b                                                                                         "
+        - "L1.12[400,500]           |L1.12-|                                                                        "
+        - "L1.13[600,700]                           |L1.13-|                                                        "
         - non_overlap
-        - "L1, all files 1b                                                                                    "
-        - "L1.11[250,350]      |------------------------------------L1.11-------------------------------------|"
+        - "L1, all files 1b                                                                                         "
+        - "L1.11[250,350]           |L1.11-|                                                                        "
```
Comment (NGA-TRAN) [inline compactor2/src/components/files_split/non_overlap_split.rs:250]: This L1.11 is definitely easier for us to review

**Review by NGA-TRAN — APPROVED** [review]
Thanks @alamb for continuing making the visual output easier for us to review. I know this is not UI tool and no matter how much we adjust, there is some side effect.

And yes, please do not merge it until  #7079 is merged 🙂

**Comment by alamb** [comment]
This PR has bitrotted and After the changes to times used in new tests (like 0-->1000000)  in certain tests, this PR would need non trivial rework. Closing

**Pull request closed without merging**